### PR TITLE
UIEH-595: fix incorrect warning message on delete resource

### PR DESCRIPTION
**Pull request by MaksymDryha**
Currently, when trying to remove a title, a warning message is always saying that we are going to delete the last title in the package, even if there are other titles in the package. The PR fixes this behavior.
[Related story](https://issues.folio.org/browse/UIEH-595)

### CODE REVIEW

**Comment by cherewaty** [comment]
Could we add a test covering this?

**Comment by MaksymDryha** [comment]
> Could we add a test covering this? 
Done!

File: test/bigtest/tests/custom-resource-edit-selection-test.js [hunk 1]
```diff
@@ -57,14 +56,38 @@ describe('CustomResourceHoldingSelection', () => {
...
       });
 
-      it('shows the confirmation modal', () => {
-        expect(ResourceEditPage.modal.isPresent).to.equal(true);
+      describe('when deselecting the last title in the package', () => {
+        beforeEach(function () {
+          providerPackage.titleCount = 1;
+          this.visit(`/eholdings/resources/${resource.titleId}/edit`);
+
+          return ResourceEditPage
```
Comment (Yurii-Danylenko) [inline test/bigtest/tests/custom-resource-edit-selection-test.js]: Please, remove return and add async await construction for consistency

File: test/bigtest/tests/resource-deselection-test.js [hunk 2]
```diff
@@ -46,13 +46,29 @@ describe('ResourceDeselection', () => {
...
+            expect(ResourcePage.deselectionModal.hasDeselectTitleWarning).to.be.true;
+          });
+        });
+      });
+
+      describe('deselecting custom title', () => {
+        beforeEach(function () {
+          title.isTitleCustom = true;
+          this.visit(`/eholdings/resources/${resource.id}`);
+          return ResourcePage.dropDownMenu.clickRemoveFromHoldings();
```
Comment (Yurii-Danylenko) [inline test/bigtest/tests/resource-deselection-test.js]: same as above

File: test/bigtest/interactors/resource-edit.js [hunk 3]
```diff
@@ -38,6 +40,7 @@ import Datepicker from './datepicker';
 }
 
 @interactor class ResourceEditDropDownMenu {
+  clickRemoveFromHoldings = clickable('.tether-element [data-test-eholdings-remove-resource-from-holdings]');
```
Comment (Yurii-Danylenko) [inline test/bigtest/interactors/resource-edit.js]: Is there another attribute "data-test-eholdings-remove-resource-from-holdings"? Why it is needed to search it inside `.tether-element`?

File: test/bigtest/tests/custom-resource-edit-selection-test.js [hunk 4]
```diff
@@ -57,14 +56,38 @@ describe('CustomResourceHoldingSelection', () => {
...
-      beforeEach(() => {
-        return ResourcePage
-          .dropDown.clickDropDownButton()
-          .dropDownMenu.clickRemoveFromHoldings();
+      describe('when the package has more than 1 title', () => {
+        beforeEach(function () {
+          providerPackage.titleCount = 20;
+          this.visit(`/eholdings/resources/${resource.titleId}/edit`);
+
+          return ResourceEditPage
```
Comment (Yurii-Danylenko) [inline test/bigtest/tests/custom-resource-edit-selection-test.js]: Please, remove return and add async await construction for consistency

**Pull request merged**